packages/lib/src/components/GooglePay/GooglePay.test.ts

```diff
@@ -55,6 +55,21 @@ beforeEach(() => {
 });
 
 describe('GooglePay', () => {
+    describe('isExpress flag', () => {
+        test('should add subtype: express when isExpress is configured', () => {
+            const googlepay = new GooglePay(global.core, { isExpress: true });
+            expect(googlepay.data.paymentMethod).toHaveProperty('subtype', 'express');
+        });
+        test('should not add subtype: express when isExpress is omitted', () => {
+            const googlepay = new GooglePay(global.core);
+            expect(googlepay.data.paymentMethod).not.toHaveProperty('subtype', 'express');
+        });
+
+        test('should throw error when express callbacks are passed but isExpress flag is not set', () => {
+            expect(() => new GooglePay(global.core, { paymentDataCallbacks: { onPaymentDataChanged: jest.fn() } })).toThrow();
+        });
```

Comment:
It would be great to see this documented somewhere. There's no mention about it on your documentation.

Reply:
Hi @asumaran - we do mention in the [release notes](https://github.com/Adyen/adyen-web/releases/tag/v6.0.0), but indeed we are not mentioning it in the payment method page itself. I shared the feedback with the docs team. Thanks for flagging it